myapp/models.py

```diff
@@ -0,0 +1,79 @@
+from datetime import datetime
+from sqlalchemy import (Column, Integer, String,
+                        DateTime, Text, ForeignKey, Enum)
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import relationship
+import enum
+#This file contains all the Flask-SQLAlchemy classes
+Base = declarative_base()
+
+
+class IdPrimaryKeyMixin(object):
+    id = Column(Integer, primary_key=True)
+
+
+class DateTimeMixin(object):
+    created_on = Column(DateTime, default=datetime.now)
+    updated_on = Column(DateTime, default=datetime.now, onupdate=datetime.now)
+
+
+# class Person(Base, IdPrimaryKeyMixin, DateTimeMixin):
```

Comment:
I don't think there's much point in merging commented out code that we don't intend to use in the project. If we want this file here as a place holder, I'd approve this PR if you removed the commented out models and views. On the other hand, if there's a good argument for including them, I am of course open to being persuaded :)